package.json

```diff
@@ -101,13 +101,15 @@
   "pnpm": {
     "overrides.csstype": "To fix type check error https://github.com/JedWatson/react-select/issues/5952",
     "overrides.mjml": "Our patch removes html-minifier from mjml-core, we dont need this dependency and its causing false-alarm (CVE https://nvd.nist.gov/vuln/detail/cve-2022-37620)",
+    "overrides.ip": "There is no update with fix for ip package, we use fork https://github.com/indutny/node-ip/issues/150#issuecomment-2325961380",
     "overrides": {
       "csstype": "3.1.2",
       "mjml-core@4.14.0>html-minifier": "-",
       "mjml-cli@4.14.0>html-minifier": "-",
       "ws@^8.0.0": "^8.18.0",
       "ws@*": ">=8.18.0 || >=7.5.10 || >=6.2.3 || >=5.2.4",
-      "cookie@<0.7.0": "0.7.2"
+      "cookie@<0.7.0": "0.7.2",
+      "ip": "https://registry.npmjs.org/neoip/-/neoip-2.1.0.tgz"
```

Comment:
just copied from github comment, you want like this?
```suggestion
      "ip": "npm:neoip@2.1.0"
```

Reply:
I think this way it's better (to point to a version and package, instead of a URL)

Reply:
changed, this syntax should work according to their docs https://pnpm.io/package_json#pnpmoverrides
<img width="481" alt="image" src="https://github.com/user-attachments/assets/dd434f88-b481-4146-848e-a173a25a6fce" />